### PR TITLE
Organization API support

### PR DESCRIPTION
**Pull request by davidcorbin**
I created two methods that I needed for a project I'm working on. I plan on adding support for the rest of the [Organization API](https://developer.github.com/v3/orgs/) on the weekend.

### CODE REVIEW

File: src/github.js [hunk 1]
```diff
@@ -289,6 +289,27 @@
          };
       };
 
+      // Organization API
+      // =======
+
+      Github.Organization = function (org) {
```
Comment (AurelioDeRosa) [inline src/github.js]: What's the aim of the `org` parameter here? It isn't used in your code.

File: test/test.org.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      github = new Github({
+         username: testUser.USERNAME,
+         password: testUser.PASSWORD,
+         auth: 'basic'
+      });
+      org = github.getOrg();
+   });
+
+   it('should check organization membership', function(done) {
+      org.checkMembership('teenscode', 'daconex', function(err) {
```
Comment (AurelioDeRosa) [inline test/test.org.js]: Please, use some organization we have control on, such as [GitHub Tools](https://github.com/github-tools) and members of our staff.

File: test/test.org.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+         should.not.exist(err);
+         done();
+      });
+   });
+
+   it('should list organization members', function(done) {
+      org.listMembers('teenscode', function(err) {
```
Comment (AurelioDeRosa) [inline test/test.org.js]: Same here.

**Comment by AurelioDeRosa** [comment]
Thank you for this PR. I've added a few comments for your review. Also, could you update the README file as well to document these new methods?

**Comment by daconex** [comment]
@AurelioDeRosa I believe I fixed all the comments you had. I can squash all the commits into one before you merge if you'd like. :smile:

**Comment by AurelioDeRosa** [comment]
Hi @daconex.

Thank you for taking the time to update your PR. Before I merge it, there is one last change I'd like you to do. I think that a instead of `checkMembership` we name the method `isMember`. Apart from that, all looks good.

**Comment by AurelioDeRosa** [comment]
Hi @daconex.

Can you do the last change required and rebase so that I can merge your PR? Thank you.

**Comment by clayreimann** [comment]
@davidcorbin This was just released in v1.2.0

**Pull request closed without merging**